### PR TITLE
Chore setup database 162881785

### PR DESCRIPTION
**Pull request by musonant**
#### What does this PR do?
- Setup database
#### Description of Task to be completed?
- Install and setup postgres database
- Implement migrations
- Implement seedings
#### How should this be manually tested?
- After cloning repository, run `npm install`
- Run `npm run migration`
- Databases should be created
#### Any background context you want to provide?
- NA
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162881785
#### Screenshots (if appropriate)
- NA
#### Questions:`
- NA

### CODE REVIEW

File: db.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Pool } from 'pg';
```
Comment (tosmak16) [inline db.js:1]: @musonant `db.js` file name should be `config.js`.

File: server/database/config.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+
```
Comment (tosmak16) [inline server/database/config.js:1]: @musonant This should be Db.js

File: server/database/config.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      connectionString: DB_URL,
+    });
+  }
+
+  /**
+   * createTables
+   */
+  async createTables() {
+    console.log('CONNECTING TO DATABASE: DATABASE URL', DB_URL);
+    this.connection.on('connect', () => {
```
Comment (tosmak16) [inline server/database/config.js:32]: @musonant What happens if the connection fails?

**Review by tosmak16 — CHANGES_REQUESTED** [review]
Good job! Implement the requested changes.

File: server/database/config.js (hunk 3 above)
Comment (musonant) [reply server/database/config.js:32]: I will add the listener for an error 
```
this.connection.on('error', (err, client) => {
    console.error('Unexpected error', err);
    process.exit(-1);
});
```
Will that do? @tosmak16

File: server/database/config.js (hunk 2 above)
Comment (musonant) [reply server/database/config.js:1]: Oh...
I have a separate file, db.js that is responsible just for connecting the database. This config.js handles migration and seeding as well. Which is why I named it so. Is there a more suitable name I should give it? @tosmak16

**Pull request merged**